image/templates/helm/shared/templates/02-scanner-v4-08-matcher-service.yaml

```diff
@@ -0,0 +1,48 @@
+{{- include "srox.init" . -}}
+{{- if ._rox.scannerV4._matcherEnabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: scanner-v4-matcher
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "srox.labels" (list . "service" "scanner-v4-matcher") | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "service" "scanner-v4-matcher") | nindent 4 }}
+spec:
+  selector:
+    app: scanner-v4-matcher
+  clusterIP: None
+  ports:
+  - name: grpc-scanner-v4-matcher
+    port: 8443
+    targetPort: grpc
+  {{ if ._rox.scannerV4.exposeMonitoring -}}
+  - name: monitoring
+    port: {{ ._rox.scannerV4.matcher.metricsPort }}
+    targetPort: monitoring
+  {{- end }}
+
+{{- if ._rox.env.istio }}
+---
+
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: scanner-v4-matcher-internal-no-istio-mtls
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "srox.labels" (list . "destinationrule" "scanner-v4-matcher-internal-no-istio-mtls") | nindent 4 }}
+  annotations:
+    stackrox.io/description: "Disable Istio mTLS for port 8443, since StackRox services use built-in mTLS."
```

Comment:
only monitoring is diabled atm. I think we need to disable both 9090 and 8443. Same in indexer